bibliopixel/drivers/SPI/driver.py

```diff
@@ -0,0 +1,42 @@
+from .. channel_order import ChannelOrder
+from .. driver_base import DriverBase
+from ... util.enum import resolve_enum
+from . import interfaces
+from enum import IntEnum
+# SubDriver imports
+from . APA102 import APA102
+from . LPD8806 import LPD8806
+from . WS281X import WS281XSPI
+from . WS2801 import WS2801
+
+
+class SPI_LEDTYPE(IntEnum):
+    APA102 = 0
+    DOTSTAR = 0
+    WS2801 = 1
+    WS281X = 2
+    WS2812 = 2
+    WS2812B = 2
+    NEOPIXEL = 2
+    LPD8806 = 3
+
+
+SPI_DRIVERS = [
+    APA102,
+    WS2801,
+    WS281XSPI,
+    LPD8806
+]
+
+
+class SPI(DriverBase):
```

Comment:
How is "using a metaclass" simpler later!?  It ain't!

Basically, there is _no_ difference between the code in the review and the simpler code except that in your case there's this dubious class `SPI` _that is never instantiated._

Reflection will work just the same in both cases.  Or, if you can give me a code sample where they are different...?